packages/framework/src/Resources/scripts/common/components/datePicker.js

```diff
@@ -5,7 +5,7 @@
     var datePicker = function ($container) {
         $container.filterAllNodes('.js-date-picker').each(function () {
             // Loads regional settings for current locale
-            var options = $.datepicker.regional[global.locale] || $.datepicker.regional[''];
+            var options = Object.assign({}, $.datepicker.regional[global.locale] || $.datepicker.regional['']);
```

Comment:
Hello @milanStano, thank you for your PR. I have one little thing. Object.assign is not suporrted in IE. You can look at [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign). You can use [extend](https://api.jquery.com/jquery.extend/) method from jquery. When we will have babel in framework (and I hope that it will be early) we will replace jquery extend as assign.

Could you change it, please?

Reply:
IE is not supported aanymore if im not wrong

Reply:
IE 11 too?

Reply:
@pesektomas Ok i fixed it :) it is okay ?

Reply:
A lots of thanks, @stanoMilan. I think that is ok. I have to test it in real use case :smile: